### PR TITLE
Hammer Wrapper and updated sync_capsule_tools_repos_to_upgrade task

### PR DESCRIPTION
**Pull request by jyejare**
This PR includes:
1. hammer.py library containing hammer def to run hammer commands
2. Updated sync_capsule_tools_repos_to_upgrade task/def w.r.t new hammer def

### CODE REVIEW

**Comment by jyejare** [comment]
closes #269

File: automation_tools/satellite6/hammer.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import json
```
Comment (elyezer) [inline automation_tools/satellite6/hammer.py]: Add a module docstring stating what kind of functions that should be present on this module.

**Comment by elyezer** [comment]
You are duplicating code fom `automation_tools.satellite6.capsule` module, since a new hammer module is being created you should refactor that module in order to use the new utilities from the hammer module.

File: automation_tools/satellite6/capsule.py [hunk 2]
```diff
@@ -506,6 +302,7 @@ def setup_capsule_content(
     :param satellite_tools_repo_url: URL of the satellite tools repository
         which will be created
     """
+    Config()
```
Comment (elyezer) [inline automation_tools/satellite6/capsule.py]: This is not needed.

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by elyezer** [comment]
ACK

**Comment by sghai** [comment]
ACK

**Pull request merged**